packages/devextreme/testing/testcafe/update_failed_etalons.mjs

```diff
@@ -0,0 +1,91 @@
+import {
+  readdirSync, lstatSync, existsSync, copyFileSync,
+} from 'fs';
+import { join, resolve } from 'path';
+import { argv } from 'process';
+
+const PATH_TO_CURRENT_ETALONS = argv[2];
+const PATH_TO_TESTCAFE_TESTS = './testing/testcafe/tests';
+const NEW_ETALON_FILE_NAME_PATTERN = /^(?!.*(?:_mask|_diff|_etalon)\.png$).*\.png$/i;
+const ETALONS_FOLDER_NAME = 'etalons';
+
+const getAllEtalonsNames = () => {
+  const allFileNames = readdirSync(PATH_TO_CURRENT_ETALONS);
+  const newEtalons = allFileNames.filter((name) => NEW_ETALON_FILE_NAME_PATTERN.test(name));
+
+  return newEtalons;
+};
+
+const isDirectory = (fileName) => lstatSync(fileName).isDirectory();
+
+function getAllDirs(path) {
+  return readdirSync(path)
+    .map((fileName) => join(path, fileName))
+    .filter(isDirectory);
+}
+
+function processEtalonFolder(etalonFolderPath, allEtalons) {
+  const allFiles = readdirSync(etalonFolderPath);
+
+  return allFiles.some((currentFileName) => {
```

Comment:
`some` is basically used to check if we have an item passing the condition. It seems to be really hacky for me to use it for simple iteration.
I suggest u using of a usual forEach/for with an additional check inside.
Like:
```javascript
allFiles.forEach(file => {
    if (newEtalonsCount === 0) {
        return;
    }
    ...
});
```
Same for `processFolder`

Reply:
fixed